<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QPushButton" name="coneButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/new/prefix1/cone.png</normaloff>:/new/prefix1/cone.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="cylinderButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/new/prefix1/cylinder.png</normaloff>:/new/prefix1/cylinder.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>120</width>
              <height>120</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="cubeButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/new/prefix1/cube.png</normaloff>:/new/prefix1/cube.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>120</width>
              <height>120</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="sphereButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/new/prefix1/Sphere.png</normaloff>:/new/prefix1/Sphere.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pyramidButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/new/prefix1/pyramid.png</normaloff>:/new/prefix1/pyramid.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="textButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="assets.qrc">
              <normaloff>:/new/prefix1/text.png</normaloff>:/new/prefix1/text.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>250</width>
              <height>170</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Insert Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Choose Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="colorComboBox">
          <item>
           <property name="text">
            <string>Black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>White</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GhostWhite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DarkBlue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BlueViolet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Crimson</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cyan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chocolate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DarkOrange</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DarkSlateGrey</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
